src/Microsoft.Health.Fhir.Api/Features/Routing/RouteNames.cs

```diff
@@ -70,5 +70,9 @@ internal static class RouteNames
         internal const string PostSearchParameterState = "PostSearchParameterState";
 
         internal const string UpdateSearchParameterState = "UpdateSearchParameterState";
+
+        internal const string GetBulkDeleteStatusById = "GetBulkDeleteStatusById";
```

Comment:
Is it possible to replace this literal with nameof(GetBulkDeleteStatusById)?

It would be something like:
internal const string GetBulkDeleteStatusById = nameof(GetBulkDeleteStatusById);

Just suggesting that as an option to minimize possible bugs due a small typo. 

The same applies to the constant below.

Reply:
Updated the entire file.